sdk/data/azcosmos/emulator_tests.go

```diff
@@ -33,6 +33,18 @@ func newEmulatorTests(t *testing.T) *emulatorTests {
 	}
 }
 
+func newEmulatorTestsWithComputeGateway(t *testing.T) *emulatorTests {
+	envCheck := os.Getenv("EMULATOR")
+	if envCheck == "" {
+		t.Skip("set EMULATOR environment variable to run this test")
+	}
+
+	return &emulatorTests{
```

Comment:
it would be good to wire host and key information for compute gateway emulator and legacy gateway emulator in a separate file and read from that.

Reply:
This is the core file in Go, there are no config files (like json) used